src/parsers/unix_socket.cpp

```diff
@@ -107,9 +108,13 @@ unix_socket parse_unix_socket_line(const std::string& line)
 
         utils::stot(tokens[INODE], sock.inode);
 
-        if (tokens.size() > PATH)
+        for (size_t i = PATH; i < tokens.size(); ++i)
```

Comment:
Unfortunately, this doesn't handle correctly the spaces in the file name.
`/var/qmux_client_socket    401` becomes `/var/qmux_client_socket 401`.

Probably it's best to replace this with:
```
        if (tokens.size() > PATH) {
            size_t path_start = line.find(tokens[PATH]);
            sock.path = line.substr(path_start);
        }
```